github-user-card/src/components/Emoji.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Emoji = props => (
+    <span
+        className="emoji"
+        role="img"
+        aria-label={props.label ? props.label : ""}
```

Comment:
The amount of accessibility here is astounding! Nice job, I love it and I'm totally gonna steal it!